conda-recipes/pytext-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pytext-nlp" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  - url: https://github.com/facebookresearch/pytext/archive/v{{ version }}.tar.gz
+    sha256: c996c1a3bdf763927ac69a0f0c40500c4b8a1b7d2ca6aa14a46c3c0f84478431 
+    patches:
+      - 0001-setup-tensorboard-req.patch
+      - 0002-set-locale.patch
+      - 0004-Fix-docstring-help-string-for-torchscript-export.patch
+      - 0005-set-generated-model-path.patch 
+      - 0007-expandpath.patch
+      - 0008-Fix-docstring-for-export.patch
+      - 0009-Make-doc_weight-and-word_weight-optional-in-IntentSlotModel.patch
+      - 0010-PR939.patch
+      - 0011-remove-gluster-paths.patch
+      - 0012-PR930.patch
+      #revert BERT support to avoid adding new dependencies
+      #pytorch-transformers (previously pytorch-pretrained-bert) and fairseq
+      - 0013-revert-BERT-support.patch  
+      - 0014-fix-UnicodeEncodeError.patch
+
+build:
+  number: 0 
+
+requirements:
+  host:
+    - python
+    - setuptools=41*
+    - pytorch=1.2.0
+
+  run:
+    - python
+    - click=7.0
+    - future=0.17.1
+    - hypothesis=3.59.1
+    - numpy=1.16.*
+    - scipy=1.3.*
+    - onnx=1.5.0
+    - joblib=0.13.2 
+    - requests=2.22
+    - pandas=0.24.2
+    - torchtext=0.4.0
+    - typing-extensions=3.7.2
+    - pytorch=1.2.0
+    - tensorboard=1.15.* 
+    - absl-py=0.7.1 
+
+about:
+  home: https://github.com/facebookresearch/pytext 
+  license: BSD
+  license_family: BSD
```

Comment:
We need the license file included here.
It needs to point to a file inside the source package or to a license file in the recipe directory.

Reply:
oh yes, missed that.

Reply:
updated it now.